docs/integrations/skopt.rst

```diff
@@ -51,101 +106,90 @@ Everything can be inspected live.
     results = skopt.forest_minimize(objective, space, callback=[neptune_callback],
                                     base_estimator='ET', n_calls=100, n_random_starts=10)
 
-Log all results
----------------
-You can log additional information from skopt results after the sweep has completed.
-By running:
+Step 5: Monitor your Skopt tuning in Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Now you can switch to the Neptune tab which you had opened previously to watch the tuning live!
```

Comment:
Could you add a screenshot or super short (5-10sec max) gif here -> it will make this more attractive and interesting for folks.

Reply:
I think the current screenshot shows "too much" with the figures and all.